docusaurus/docs/Android/04-compose/03-channel-components/04-default-channel-item.mdx

```diff
@@ -0,0 +1,203 @@
+        }
+    }
+}
+```
+
+In the example above, we customized the `onChannelLongClick`  and `onChannelClick` action handlers, to open the messages screen when the user taps on an item, or to store the selected channel state in the `ViewModel`, on long taps.
+
+This way, you get more control over what happens when the user interacts with the items, but you still have the default UI that you don't have to implement.
+
+Read on to learn how to customize the UI.
+
+## Customization
+
+If you're looking to customize the UI of the `DefaultChannelItem`, there are a few ways you can do so, as per the signature:
+
+```kotlin
+@Composable
+fun DefaultChannelItem(
+    ..., // state and action handlers
+    modifier: Modifier = Modifier,
+    leadingContent: @Composable RowScope.(Channel) -> Unit = {
+        ChannelAvatar(
+            modifier = Modifier
+                .padding(horizontal = 8.dp)
+                .size(40.dp),
+            channel = it,
+            currentUser = currentUser
+        )
+    },
+    detailsContent: @Composable RowScope.(Channel) -> Unit = {
+        ChannelDetails(
+            channel = it,
+            currentUser = currentUser,
+            modifier = Modifier
+                .weight(1f)
+                .wrapContentHeight()
+        )
+    },
+    trailingContent: @Composable RowScope.(Channel) -> Unit = {
+        ChannelLastMessageInfo(
+            channel = it,
+            currentUser = currentUser,
+            modifier = Modifier
+                .padding(horizontal = 8.dp)
+                .wrapContentHeight()
+                .align(Alignment.Bottom)
+        )
+    },
+    divider: @Composable ColumnScope.() -> Unit = {
+        Spacer(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(0.5.dp)
+                .background(color = ChatTheme.colors.borders)
+        )
+    },
+)
+```
+
+* `modifier`: The modifier parameter for the root component. You can apply a background, elevation, padding, shape, touch handlers and much more.
+* `leadingContent`: Customizable composable function that allows you to override the content that appears at the start of the list item. By default, it represents a `ChannelAvatar`.
+* `detailsContent`: Customizable composable function that allows you to override the center part of the list item. By default, it represents the `ChannelDetails` that show its name and the last message preview.
+* `trailingContent`: Customizable composable function that allows you to override the content that appears at the end of the list item. By default, it represents the `ChannelLastMessageInfo` that shows the unread message indicator if there are unread messages and the last message read state.
+* `divider`: Customizable composable function that allows you to override the content that appears at the bottom of the list item, as a divider.
+
+Here's a simple example for building your own channel item, by overriding the `mentioned` parameters:
```

Comment:
Why "mentioned" is surrounded with backticks?

Reply:
I think I was going to mention a specific param and then accidentally put this in